dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/helpers/validate-csv-cell-data/index.test.ts

```diff
@@ -0,0 +1,59 @@
+import { UTILISATION_REPORT_HEADERS } from '@ukef/dtfs2-common';
+import validateUtilisationReportCsvCellData from '.';
+import {
+  generateUkefFacilityIdError,
+  generateBaseCurrencyError,
+  generateFacilityUtilisationError,
+  generatePaymentCurrencyError,
+  generatePaymentExchangeRateError,
+} from '../..';
+
+jest.mock('../..', () => ({
+  generateUkefFacilityIdError: jest.fn(),
+  generateBaseCurrencyError: jest.fn(),
+  generateFacilityUtilisationError: jest.fn(),
+  generatePaymentCurrencyError: jest.fn(),
+  generatePaymentExchangeRateError: jest.fn(),
+  generateTotalFeesAccruedCurrencyError: jest.fn(),
+  generateTotalFeesAccruedExchangeRateError: jest.fn(),
+}));
```

Comment:
Do we need to mock these? I would expect an export and test for `rowValidations` and then this to be consumed in the `validateUtilisationReportCsvCellData` unit test. Not sure on the existing patterns though.

Reply:
I am not sure, this was already there and I moved it.  I believe it is needed to mock the mongoDB call

Reply:
Happy to resolve purely because it's only being moved in this PR